CyborgStation Submission

CircuitCyborg opened the pull request.

CircuitCyborg committed:
Resubmitting to correct repo

qcoral reviewed (changes requested):
this is awesome!!! couple things
- could i get an itemizted breakdown of what parts will cost what so i can keep track of your grant
- can you also post your schematic / a bit more info on what your weather station does?? tysm!!

CircuitCyborg committed:
Update readme_+_BOM.md

Added links and itemized costs

CircuitCyborg committed:
Update to readme_+_BOM.md

Added an image of the schematic plus a description of what the device will do and some plans for future software versions.

CircuitCyborg commented:
Added requested changes

CircuitCyborg committed:
Update readme_+_BOM.md

Reupdated BOM

qcoral reviewed (approved):
LGTM!

congrats - get your weather station here:

forms.hackclub.com/asylum
